### PR TITLE
feat: Send facilitator requests sooner

### PR DESCRIPTION
**Pull request by smockle**
This PR updates the logic that filters meeting issues, so that issues up to 1 week out (previously: 1 day out) are now included when determining facilitator needs.

Specifically, the `getIsTomorrow` function is renamed to `getIsSoon`, and callsites in action and test code are updated accordingly.

### CODE REVIEW

**Review by inkblotty — COMMENTED** [review]
Can we only get these changes to apply to Office Hours and not Deep Dives? 🙏

**Comment by smockle** [comment]
> Can we only get these changes to apply to Office Hours and not Deep Dives? 🙏

Done! https://github.com/inkblotty/daily-update-action/pull/39/commits/6be049190979ef90048aace4406afe95560e2fdd

I refactored `getIsTomorrow` and `getIsSoon` into a new function: `getIsWithinDays`. Now, Office Hours call `getIsWithinDays(date, 7)`, and Deep Dives call `getIsWithinDays(date, 1)`[^1].

While I was testing[^2], I caught an unexpected issue: The test `Deep Dives › getAndMapDeepDiveIssues › has expected length and first option is high priority` consistently fails. I switched to `main`, and discovered the test is failing there too.

[^1]: It’s not an _elegant_ function signature. I considered adding aliases, e.g. `export const getIsTomorrow = (x) => getIsWithinDays(x, 1)`.
[^2]: `npm ci && npm run build && npm run test`

File: src/shared.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const formatDate = (date: Date): string => {
 
 export const oneDayMs = 1000 * 60 * 60 * 24;
 
-export const getIsTomorrow = (date: Date): boolean => {
+export const getIsWithinDays = (date: Date, days: number): boolean => {
```
Comment (inkblotty) [inline src/shared.ts:24]: Nit: Can we call this `getIsWithinFutureDays` since it will return false for yesterday?

**Review by inkblotty — APPROVED** [review]
LGTM with name change

File: src/shared.ts (hunk 1 above)
Comment (smockle) [reply src/shared.ts]: Done! https://github.com/inkblotty/daily-update-action/pull/39/commits/c4b852676bae9ed7e329c97f62542c8d359fa457

**Pull request merged**